.github/workflows/cd_gestor_rutas_entrega.yml

```
@@ -0,0 +1,98 @@
+
+      - name: Desplegar en GKE
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, 'CD')
+        run: |
+          kubectl apply -f backend/k8s/gestor_rutas_entrega_deployment.yaml
+          kubectl apply -f backend/k8s/gestor_rutas_entrega_service.yaml
+
+      - name: Verificar rollout exitoso gestor_rutas_entrega
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, 'CD')
+        run: |
+          kubectl rollout status deployment/gestor_rutas_entrega
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)